README.md

```diff
@@ -96,21 +103,32 @@ Further, we realized that this would allow us to build the terminal's renderer a
 
 ## FAQ
 
-## I built and ran the new Terminal, but I just get a blank window app!
+## I built and ran the new Terminal, but I just get a blank window
+
+Cause: Your locally built Terminal does not match the architecture of your computer.
+
+Solution: Set Visual Studio's "Solution Platform" to match your computer's architecture, then build and run the CascadiaPackage solution.
 
-Make sure you are building for your computer's architecture. If your box has a 64-bit Windows, change your Solution Platform to x64.  
-To check your OS architecture go to Settings -> System -> About (or Win+X -> System) and under `Device specifications` check for the  `System type`.
+![Visual Studio's Solution Platform](/doc/images/solution-platform.png)
+
+> 👉 Tip: To check your computer's architecture, open Windows 10's Settings app and go to System -> About (or Win+X -> System) and under `Device specifications` check for the  `System type`.
```

Comment:
This entire section has been replaced with a message box that appears during startup if you run the wrong architecture. You can remove it!

Reply:
Thanks. Done.